styles.css

```diff
@@ -173,7 +173,6 @@ body {
 
 #footer .footer_icon-links .footer_icon .footer_icon-img {
   height: 1.8rem;
-  filter: invert(100%);
 }
 
```

Comment:
- [ ]  Great work on your portfolio but there are some aspects you can improve on.
 - In your contact section, you can increase the padding around the placeholder text inside the `<textarea>` element, especially from the left.  Adding padding around the placeholder text can make it easier to read and understand. Without padding, the placeholder text can appear cramped or too close to the edge of the input field, making it harder to distinguish from other text.
 
![image](https://user-images.githubusercontent.com/98453979/230596503-0a33c605-f0d8-4b51-864c-ea9341414dae.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/230596586-ab28072c-5db0-4a4a-9138-5b34801a3557.png)

- [ ] Kindly add a thin gray line that separates the footer from the contact section. This will create a more balanced and visually appealing design.

![image](https://user-images.githubusercontent.com/98453979/230597177-0dfd7faa-21a3-44cc-9eaf-500f5ae24118.png)